code/init.dm

```diff
@@ -1,4 +1,4 @@
-//init file stolen from hippie
+//init file stolen from hippie but the real question is why
 
 /proc/cit_initialize()
```

Comment:
Hold on, just move the contents to the game/world.dm and remove this proc and the file. It's just mentors and loadout items after all.

Reply:
i moved it to the same place as admin/donator loadout loading so it should be fine now